osquery/sql/sqlite_util.cpp

```diff
@@ -118,11 +124,23 @@ SQLiteDBInstance::~SQLiteDBInstance() {
 
 void SQLiteDBManager::unlock() { instance().lock_.unlock(); }
 
+bool SQLiteDBManager::isDisabled(const std::string& table_name) {
+  const auto& element = instance().disabled_tables_.find(table_name);
+  return (element != instance().disabled_tables_.end());
+}
+
+std::unordered_set<std::string> SQLiteDBManager::parseDisableTablesFlag(std::string list) {
+  const auto& tables = split(list, ",");
+  return std::unordered_set<std::string>(tables.begin(), tables.end());
+}
+
 SQLiteDBInstance SQLiteDBManager::getUnique() { return SQLiteDBInstance(); }
 
 SQLiteDBInstance SQLiteDBManager::get() {
   auto& self = instance();
 
+  self.disabled_tables_ = SQLiteDBManager::parseDisableTablesFlag(FLAGS_disable_tables);
```

Comment:
Move this to: https://github.com/sharvilshah/osquery/blob/disable_tables_runtime_flag/osquery/sql/sqlite_util.h#L109
and you'll only parse the flags once. 

The way it's called now, inside of `::get`, the tokenization will happen every time the SQLite DB is accessed. 

You can also call:

```cpp
self.disabled_tables_ = self.parseDisableTablesFlag(FLAGS_disable_tables);
```

And remove the static below.